socialite/src/components/StackOverflow/StackOverflow.js

```diff
@@ -107,21 +117,11 @@ function StackOverflow(props){
                         </div>
                         <div className="top-users m-4">
                           <ul>
-                            <li className="mt-4">Prajneya
-                              <div className="no-post float-right">231 posts</div>
-                            </li>
-                            <li className="mt-4">Prajneya
-                              <div className="no-post float-right">231 posts</div>
-                            </li>
-                            <li className="mt-4">Prajneya
-                              <div className="no-post float-right">231 posts</div>
-                            </li>
-                            <li className="mt-4">Prajneya
-                              <div className="no-post float-right">231 posts</div>
-                            </li>
-                            <li className="mt-4">Prajneya
-                              <div className="no-post float-right">231 posts</div>
+                          {people_list && people_list.map(person => ( 
+                            <li className="mt-4">{person['username']}
+                              <div className="no-post float-right">{person['times_answered']} posts</div>
```

Comment:
Same here. It would be better if "posts" is changed to answers as we are calling questions as "posts".

Reply:
Will fix in the revamp PR.